modules/javafx.base/src/test/java/test/javafx/collections/ModifiableObservableListBaseTest.java

```diff
@@ -204,15 +204,15 @@ public void testInvalidIndexThrowsIOOBE() {
             var nonEmptySubList = new MockModifiableObservableList(
                     new ArrayList<>(List.of("a", "b", "c")),
                     new ArrayList<>(List.of("a", "b")))
-                .subList(0, 2);
+                    .subList(0, 2);
```

Comment:
There are only whitespace changes in this file, since it was already converted to JUnit 5. I recommend to revert your changes.